design/features/routing/README.md

```diff
@@ -1,40 +1,78 @@
-# Simple routing logic (skip questions)
+# Routing 
 
-## Status
+## Status 
 
-Date created: *2023-05-23*  
-
-Epic trello card: https://trello.com/c/nBnFZQlw/135-epic-simple-routing  
-Mural working board: https://app.mural.co/invitation/mural/gaap0347/1671196865223?sender=ue1ef9fc3c2ad3697c3c93132&key=09616cf9-ae28-43a7-8810-10970a59a765  
-Basic routing feature write up: https://docs.google.com/document/d/1i7T5D3PQw4hB1wgISi-r5086Fzw1-3nKnwaCGbNMi1E/edit#heading=h.h2rnqdwxl250  
-___
+Date updated: *2024-10-31*  
 
 ## Contents
 
+- [Branching and exit pages](#branching-and-exit-pages-iteration-1)
+  - [What](#what)
+  - [Why](#why)
+  - [Hypothesis](#hypothesis)
+
 - [Simple routing logic (skip questions)](#simple-routing-logic-skip-questions)
-  - [Status](#status)
-  - [Contents](#contents)
   - [What](#what)
   - [Why](#why)
   - [Hypothesis](#hypothesis)
 
 ___
 
-<br>
+## Branching and exit pages (iteration 1)
+
+Date updated: *2024-10-24* 
+
+Epic trello card: https://trello.com/c/a5c3JaWu/1908-epic-branching-and-exit-pages  
+Mural working board: https://app.mural.co/t/gaap0347/m/gaap0347/1728478914347/6a7e0b709f50f00f81cab37414fdf44ec8601a70?wid=0-1729257264973  
+Routing and branching terminology (how we decided what to deliver): https://docs.google.com/document/d/1lXKKGyT6kbFc0EbthTPiEHK5sEIc1EuVqSLXuQ1MXPc/edit?usp=sharing  
+
+### What  
+
+This feature introduces new functionality to the form building tool to help form creators create simple skip logic for questions. This means you can skip someone to a later question in a form based on their response to a question where they have to select one answer from a list.
+
+#### User stories
+
+**As a form creator**, I need to allow users to skip to the relevant group of questions based on their previous answer so that we only collect the information needed.  
+
+**As a form filler**, I need to skip one or more questions based on my previous answer so that I don’t have to give information that is not relevant to me.  
+
+**As a form processor**, I need to receive the information when people have taken different branches in a way that I can easily add it up.  
+
+
+### Why  
+
```

Comment:
Should we add something here? The 'Simple routing logic' section lower down has a 'Why' and a 'Hypothesis' section...

Reply:
Because it’s an H3 under the H2 it should be fine, but it would maybe be helpful. I just couldn’t think of anything useful to add. 
Would also need to add to the “what” and also for the simple routing bits (why and what)

Reply:
Aren't they both H3s though? I can have a think about this next week if needed but have run out of time this week :(

Reply:
Yeah both why’s and what’s are H3s, but they each sit under a unique H2 (the name of the feature, eg simple routing). So programmatically it reads as though it’s within the feature. 

But if a screen reader goes through then I agree they might not get ALL the context depending on how they’ve navigated. If you have time to think of something that’d be great - I might take a look before next week.